CHANGELOG.md

```diff
@@ -10,6 +10,20 @@
 
 ### Features
 
+- Add RNSentrySDK APIs support to @sentry/react-native/expo plugin ([#4633](https://github.com/getsentry/sentry-react-native/pull/4633))
+
+  This feature is opt-out to enable it set `useNativeInit` to `true` in your `@sentry/react-native/expo` plugin configuration.
```

Comment:
NIT: It may make it easier for reading.
```suggestion
  This feature is opt-out, to enable it set `useNativeInit` to `true` in your `@sentry/react-native/expo` plugin configuration.